### ISSUE TITLE
Update weather module tests

### ISSUE THREAD

**Issue by Androbin** [apertium/phenny#514]
```
======================================================================
FAIL: test_locations (phenny.modules.test.test_weather.TestWeather)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/apertium/phenny/web.py", line 62, in wrapper
    return fn(*args, **kw)
  File "/home/travis/build/apertium/phenny/modules/test/test_weather.py", line 47, in test_locations
    check_location(result, expected)
  File "/home/travis/build/apertium/phenny/modules/test/test_weather.py", line 20, in check_location
    self.assertAlmostEqual(result[0], expected[0], places=1)
AssertionError: 41.7533819 != 41.7 within 1 places (0.053381899999997984 difference)
```

**Comment by Androbin** [comment]
It's so cumbersome to constantly have to update these. We really need a better solution.

**Comment by tomasff** [comment]
@Androbin I created a pull request to fix this issue. I explain my reasoning in there too.

**Comment by Androbin** [comment]
Closed by #521
